src/main/java/dao/ReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package dao;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
+    private static final String FILE_PATH = "src/main/resources/input.csv";
+
+    @Override
+    public List<String> read() {
```

Comment:
```suggestion
    public List<String> read(String fileName) {
```
Reader and writer should take path as an input, this way code can be reused